packages/kbn-rule-data-utils/src/technical_field_names.ts

```diff
@@ -34,6 +34,7 @@ const ALERT_EVALUATION_THRESHOLD = `${ALERT_NAMESPACE}.evaluation.threshold` as
 const ALERT_EVALUATION_VALUE = `${ALERT_NAMESPACE}.evaluation.value` as const;
 const ALERT_ID = `${ALERT_NAMESPACE}.id` as const;
 const ALERT_OWNER = `${ALERT_NAMESPACE}.owner` as const;
+const ALERT_CONSUMERS = `${ALERT_NAMESPACE}.consumers` as const;
```

Comment:
Are these going to be shared? There are a bunch of fields in `security_solution`, not in here and I'm just trying to figure out whether they should be added here or just kept within security_solution i.e. all the `original_event` fields here: https://github.com/elastic/kibana/blob/b5e8db2443fc882a83ca6a5297e9ed55d0116efa/x-pack/plugins/security_solution/public/common/components/drag_and_drop/helpers.ts#L113

Reply:
Yeah the fields here are expected to be used by all solutions that use alerts as data. The `original_event` fields are security-specific because other solutions are not attempting to preserve the whole source documents (so they don't need to preserve the original `event.*` fields when they populate fields like `event.kind: signal`)

Reply:
👍🏾

Reply:
I'm not sure. For now I just made it similar to the existing `ALERT_PRODUCER`. I think as long as we're using the constants from this package, follow up naming discussions can be had to change the field names as needed.

Reply:
I'm just looking at the spec and I can't find `kibana.alert.consumers` there, but:

![](https://puu.sh/I1Kyv/5967a3e28a.png)

I think it's `kibana.consumers` because the concept of shareability between features might be more broad than the alerting domain.

Not something that this PR should be responsible for, so I think we can merge it as is, but I'd double-check this naming at the next alerts-as-data meeting.